calculateDiscount.js

```diff
@@ -0,0 +1,27 @@
+let array=[]
```

Comment:
You do not have to declare this variable, just use the array data directly when calling your function.